### PR TITLE
TINY-2293: add auto test script for context menu on link

### PR DESCRIPTION
**Pull request by ArvinJ-H**
Related Ticket: TINY-2293

Description of Changes:
Add context menu check for link plugin

Pre-checks:
* [x] ~Changelog entry added~
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/` for new features (if applicable)

Review:
* [x] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

File: modules/tinymce/src/plugins/link/test/ts/browser/LinkContextMenuTest.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    toolbar: 'link',
+    base_url: '/project/tinymce/js/tinymce',
+  }, [ Plugin ]);
+
+  const pAssertFocusOnItem = (label: string, selector: string) =>
+    FocusTools.pTryOnSelector(`Focus should be on: ${label}`, SugarDocument.getDocument(), selector);
+
+  const pressDownArrowKey = (editor: Editor) => TinyUiActions.keydown(editor, Keys.down());
+
+  it('TBA: Context menu show up on link', async () => {
```
Comment (lnewson) [inline modules/tinymce/src/plugins/link/test/ts/browser/LinkContextMenuTest.ts:21]:
```suggestion
  it('TINY-2293: Context menu show up on link', async () => {
```
We normally try to include the Jira number so we have something to track back to why this was added.

**Review by lnewson — APPROVED** [review]

**Comment by lnewson** [comment]
One thing to note is that we need the PR title updated to be something descriptive, as the title will be used for the final commit message.

- [ ] Fix PR title

File: modules/tinymce/CHANGELOG.md [hunk 2]
```diff
@@ -82,6 +82,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Deprecated
 - The dialog button component `primary` property has been deprecated in favour of the new `buttonType` property #TINY-8304
 
+
+## 6.0.0 - 2022-01-20
+### Added
+- Auto test script for context menu check on links
```
Comment (lnewson) [inline modules/tinymce/CHANGELOG.md:88]: This shouldn't be here as it's not customer facing... it's also in the wrong place as change for 6.0 should be in the `Unreleased` section.
```suggestion
```

**Review by lnewson — APPROVED** [review]

File: modules/tinymce/CHANGELOG.md [hunk 3]
```diff
@@ -82,6 +82,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Deprecated
 - The dialog button component `primary` property has been deprecated in favour of the new `buttonType` property #TINY-8304
 
+
```
Comment (lnewson) [inline modules/tinymce/CHANGELOG.md:85]:
```suggestion
```

**Review by lnewson — APPROVED** [review]

**Review by TheSpyder — APPROVED** [review]

**Pull request merged**